geminimd/drive.py

```
@@ -19,13 +21,35 @@
 SCOPES = ["https://www.googleapis.com/auth/drive.readonly"]
 TOKEN_FILE = "token.json"
 
+def _retry(fn, *, retries: int = 3, base_delay: float = 0.5):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_retry`

(ANN202)